### ISSUE TITLE
Image fields: consider allowing .jpe and other variants as possible extensions for the image/jpeg mime type

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5108]
While filing #5107, I noticed that Drupal includes this extension in their "Convert" effect for image styles:

<img width="50%" height="50%" src="https://user-images.githubusercontent.com/2423362/119925754-b8ac0d80-bf7e-11eb-9524-e0b3bc3bfec0.png">

Turns out that this is actually a thing: https://www.keycdn.com/support/difference-between-jpg-and-jpeg

> The MIME media type for JPEG is image/jpeg (defined in [RFC 1341](https://www.ietf.org/rfc/rfc1341.txt)), except in older Internet Explorer versions, which provides a MIME type of image/pjpeg when uploading JPEG images. JPEG is also defined with the additional extensions, `.jpe`, `.jif`, `.jfif`, and `.jfi`.

Here's the d.org issue that resulted in adding that extension: [GDToolkit::getSupportedExtensions returns incomplete list](https://www.drupal.org/project/drupal/issues/2477381) ...and the respective commit: https://git.drupalcode.org/project/drupal/-/commit/2d292cd1ef7e3c752c65960d656461913b8e6540

Related: https://stackoverflow.com/questions/33692835/is-the-mime-type-image-jpg-the-same-as-image-jpeg

**Comment by klonos** [comment]
To be clear: I realize that .jpe and the rest of the variants aren't as popular as .jpg and .jpeg + there has been no complaint in the past years re anyone missing this functionality. So I don't have particularly strong feelings of whether we should implement this or not. My thinking is that if it only takes a relatively small change to save time/effort for people using Backdrop, then we should do it.

**Comment by BWPanda** [comment]
I've never even heard of this. If it was a new format with improvements over .jpg (like .webp seems to be) then I'd say yes. But it looks like something that's been around for years, no one's heard of, so why bother?

**Comment by klonos** [comment]
>so why bother?

Yeah 🤔 ...so the JPEG and the JPG extensions that are on that dropdown in the screenshot are basically the same mime type as JPE. So say we are to implement #5107, people coming from Drupal-land might wonder why we "are missing a format". The only reason I raised this is for parity.
